maserol/preprocess.py

```diff
@@ -83,8 +83,8 @@ def assembleKav(data: xr.DataArray, ab_types: Optional[Iterable]=DEFAULT_AB_TYPE
         raise ValueError(f"Invalid receptors passed into assembleKav: {','.join(other)}. Receptors must be FCRs or IGGs.")
 
     # assemble matrix
-    Kav = xr.DataArray(np.full((len(igg+fc), len(ab_types)), 10),
-                       coords=[igg+fc, list(ab_types)],
+    Kav = xr.DataArray(np.full((len(receptors), len(ab_types)), 10),
+                       coords=[receptors, list(ab_types)],
```

Comment:
Hmm, so `assembleKav()` won't be able to select only the entries we need? We can assume/enforce all data must go through `prepare_data` before analysis.

Reply:
We select the receptors we want in prepare_data, so we can just copy the list of receptors from `cube`. This way we don't have to code to reorder the receptors in two places.